mllib/src/main/scala/org/apache/spark/ml/param/shared/SharedParamsCodeGen.scala

```diff
@@ -95,7 +95,9 @@ private[shared] object SharedParamsCodeGen {
       ParamDesc[String]("distanceMeasure", "The distance measure. Supported options: 'euclidean'" +
         " and 'cosine'", Some("org.apache.spark.mllib.clustering.DistanceMeasure.EUCLIDEAN"),
         isValid = "(value: String) => " +
-        "org.apache.spark.mllib.clustering.DistanceMeasure.validateDistanceMeasure(value)")
+        "org.apache.spark.mllib.clustering.DistanceMeasure.validateDistanceMeasure(value)"),
+      ParamDesc[String]("validationIndicatorCol", "the indicator column name for indicating " +
```

Comment:
How about rephrasing the description: "name of the column that indicates whether each row is for training or for validation.  False indicates training; true indicates validation."